subprojects/gradle/upload-to-googleplay/build.gradle.kts

```diff
@@ -1,13 +1,13 @@
 plugins {
-    id("kotlin")
-    `maven-publish`
-    id("com.jfrog.bintray")
+    id("com.avito.android.kotlin-jvm")
+    id("com.avito.android.publish-kotlin-library")
+    id("com.avito.android.libraries")
 }
```

Comment:
I'm thinking of common naming patterns for convention plugins:
- To distinguish easier plugins from buildSrc from "external" plugins
- To understand what it applies behind

As I understand, the package name `com.avito.android` is used in public repositories to avoid clashing. In convention plugins, we have more control and can omit it.

Possible options:
- `id(convention.kotlin-jvm)`
- `id(convention.com.android.library)`
- ...

What do you think?

Reply:
yes, i like this idea; but let me rename it afterwards

Reply:
ok